Google.Api.Gax/FieldGenerators.cs

```diff
@@ -0,0 +1,27 @@
+﻿/*
+ * Copyright 2024 Google LLC
+ * Use of this source code is governed by a BSD-style
+ * license that can be found in the LICENSE file or at
+ * https://developers.google.com/open-source/licenses/bsd
+ */
+
+using System;
+
+namespace Google.Api.Gax;
+
+// TBD:
+// - Class name
+// - Method names
+// - Package
+// Basically everything!
+
+/// <summary>
+/// Generator methods for auto-populated fields.
+/// </summary>
+public static class FieldGenerators
```

Comment:
`FieldValueGenerators` or `ValueGenerators` ? I think just Field is confusing, specially when quickly seen in generator code.

Is something other than value generation that we expect to be doing here? Validations are unlikely I guess. But maybe canonicalization? Just thinking whether we want *Generators and the hypotethetical *Canonicalizations or simply *Helpers. (* is Field or FieldValue or Value).

Reply:
Changed to FieldFormats.